app/views.py

```diff
@@ -3,12 +3,18 @@
 import re
 from json import loads
 
-from app import app, models, sql_session, viewsModel
```

Comment:
There's a few ways you could split up views_model.py. It's quite useful to think in terms of *object orientation* here. You're pointing in the right direction, I think; coming up with some closed set of *resources* that implement a similar interface. Similar to how we discussed implementing a RESTful URL scheme.

For instance, in that module you've got some bare functions that start with `get_` and some that start with `update_`. Can you identify key resources that are the recipients of generic messages like `get` and `update`? If so, you can orient your code around them as objects. For instance, you might have a `class Video()` that has methods `def get()` and `def update()`. Presumably the same information (a table name, for instance) is in more than one of those functions; if they were methods on the same object, then they could be centralized into the object itself. You might see opportunities to create a base class that takes care of SQL queries in a more generalized way.